include/tables/scene_table.h

```diff
@@ -9,107 +9,107 @@
-/* 0x23 */ DEFINE_SCENE(shrine_scene, none, SCENE_SHRINE, SDC_DEFAULT, 0, 0)
-/* 0x24 */ DEFINE_SCENE(shrine_n_scene, none, SCENE_SHRINE_N, SDC_DEFAULT, 0, 0)
-/* 0x25 */ DEFINE_SCENE(shrine_r_scene, none, SCENE_SHRINE_R, SDC_DEFAULT, 0, 0)
+/* 0x00 */ DEFINE_SCENE(ydan_scene, g_pn_06, SCENE_YDAN, SDC_YDAN, 1, 2) // DEKU_TREE
+/* 0x01 */ DEFINE_SCENE(ddan_scene, g_pn_08, SCENE_DDAN, SDC_DDAN, 1, 3) // DODONGOS_CAVERN
+/* 0x02 */ DEFINE_SCENE(bdan_scene, g_pn_07, SCENE_BDAN, SDC_BDAN, 1, 4) // JABU_JABU
+/* 0x03 */ DEFINE_SCENE(Bmori1_scene, g_pn_01, SCENE_BMORI1, SDC_BMORI1, 2, 5) // FOREST_TEMPLE
+/* 0x04 */ DEFINE_SCENE(HIDAN_scene, g_pn_03, SCENE_HIDAN, SDC_HIDAN, 2, 6) // FIRE_TEMPLE
+/* 0x05 */ DEFINE_SCENE(MIZUsin_scene, g_pn_04, SCENE_MIZUSIN, SDC_MIZUSIN, 1, 7) // WATER_TEMPLE
+/* 0x06 */ DEFINE_SCENE(jyasinzou_scene, g_pn_05, SCENE_JYASINZOU, SDC_JYASINZOU, 1, 8) // SPIRIT_TEMPLE
+/* 0x07 */ DEFINE_SCENE(HAKAdan_scene, g_pn_02, SCENE_HAKADAN, SDC_HAKADAN, 2, 9) // SHADOW_TEMPLE
+/* 0x08 */ DEFINE_SCENE(HAKAdanCH_scene, g_pn_54, SCENE_HAKADANCH, SDC_HAKADAN, 2, 10) // BOTTOM_OF_THE_WELL
+/* 0x09 */ DEFINE_SCENE(ice_doukutu_scene, g_pn_10, SCENE_ICE_DOUKUTO, SDC_ICE_DOUKUTO, 0, 0) // ICE_CAVERN
+/* 0x0A */ DEFINE_SCENE(ganon_scene, none, SCENE_GANON, SDC_DEFAULT, 2, 0) // GANONS_TOWER
+/* 0x0B */ DEFINE_SCENE(men_scene, g_pn_11, SCENE_MEN, SDC_MEN, 0, 0) // GERUDO_TRAINING_GROUND
+/* 0x0C */ DEFINE_SCENE(gerudoway_scene, g_pn_49, SCENE_GERUDOWAY, SDC_GERUDOWAY, 0, 0) // THIEVES_HIDEOUT
+/* 0x0D */ DEFINE_SCENE(ganontika_scene, g_pn_09, SCENE_GANONTIKA, SDC_GANONTIKA, 0, 0) // INSIDE_GANONS_CASTLE
+/* 0x0E */ DEFINE_SCENE(ganon_sonogo_scene, none, SCENE_GANON_SONOGO, SDC_GANON_SONOGO, 0, 0) // GANONS_TOWER_COLLAPSE
+/* 0x0F */ DEFINE_SCENE(ganontikasonogo_scene, none, SCENE_GANONTIKA_SONOGO, SDC_GANONTIKA_SONOGO, 0, 0) // INSIDE_GANONS_CASTLE_COLLAPSE
+/* 0x10 */ DEFINE_SCENE(takaraya_scene, g_pn_51, SCENE_TAKARAYA, SDC_DEFAULT, 0, 0) // TREASURE_BOX_SHOP
+/* 0x11 */ DEFINE_SCENE(ydan_boss_scene, none, SCENE_YDAN_BOSS, SDC_YDAN_BOSS, 0, 0) // DEKU_TREE_BOSS
+/* 0x12 */ DEFINE_SCENE(ddan_boss_scene, none, SCENE_DDAN_BOSS, SDC_DEFAULT, 0, 0) // DODONGOS_CAVERN_BOSS
+/* 0x13 */ DEFINE_SCENE(bdan_boss_scene, none, SCENE_BDAN_BOSS, SDC_BDAN, 0, 0) // JABU_JABU_BOSS
+/* 0x14 */ DEFINE_SCENE(moribossroom_scene, none, SCENE_MORIBOSSROOM, SDC_DEFAULT, 1, 0) // FOREST_TEMPLE_BOSS
+/* 0x15 */ DEFINE_SCENE(FIRE_bs_scene, none, SCENE_FIRE_BS, SDC_HIDAN, 0, 0) // FIRE_TEMPLE_BOSS
+/* 0x16 */ DEFINE_SCENE(MIZUsin_bs_scene, none, SCENE_MIZUSIN_BS, SDC_MIZUSIN_BS, 0, 0) // WATER_TEMPLE_BOSS
+/* 0x17 */ DEFINE_SCENE(jyasinboss_scene, none, SCENE_JYASINBOSS, SDC_DEFAULT, 0, 0) // SPIRIT_TEMPLE_BOSS
+/* 0x18 */ DEFINE_SCENE(HAKAdan_bs_scene, none, SCENE_HAKADAN_BS, SDC_HAKADAN, 0, 0) // SHADOW_TEMPLE_BOSS
+/* 0x19 */ DEFINE_SCENE(ganon_boss_scene, none, SCENE_GANON_BOSS, SDC_DEFAULT, 0, 0) // BOSS_GANONDORF
```

Comment:
I've also thought of putting ARENA in there, since the symbol equivalent like gGanonBoss is problematic as it clashes with the object anyway, unless the scene/room symbol names also get Scene/RoomN in the prefix

Reply:
LAIR is another terms that's been thrown around for boss rooms, so `GANONDORFS_LAIR` and the like

Reply:
I would probably go with arena over lair personally, but not sure either are needed

I'm also fine with using the boss names in the names, not sure if that was avoided for a specific reason

Reply:
I named the boss scenes with a _boss suffix to mimic the debug names, and also because the two scenes are tightly linked together

Boss scenes also have minimal geometry from the dungeon itself to maintain the illusion of being in the same place when exiting boss scenes, that could be an argument towards not naming things "lair" or "arena", but I would ignore that argument

I wouldn't be opposed to using the bosses' names, though I expect them to be less consensual than a _boss prefix but don't let that prevent us from trying

Reply:
bump, can I get anyone's opinion on naming boss scenes

Reply:
Hmm, I think they are fine as is but I think adding boss names is a good idea. Arena also seems like a good idea, though if the scene symbols all get a common `Scene`, it doesn't seem necessary.

Reply:
even if "arena" is not added I'd personally still like to see `BOSS_GANONDORF` -> `GANONDORF_BOSS` and `GANON_BOSS` to more closely match the pattern of the other boss scene suggestions